packages/tasit-contracts/migrations/4_gnosis_contracts.js

```diff
@@ -0,0 +1,20 @@
+var ProxyFactory = artifacts.require("./ProxyFactory.sol");
+var GnosisSafe = artifacts.require("./GnosisSafe.sol");
+
+// accounts[9]
+const johnAddress = "0x8226bcef50b3c76a9eb7eba0c09ebbb2362e5db7";
+
+module.exports = function(deployer) {
+  // Workaround to write async/await migration scripts
+  // See more: https://github.com/trufflesuite/truffle/issues/501#issuecomment-332589663
+  deployer.then(async () => {
+    const proxyFactory = await deployer.deploy(ProxyFactory);
+    const safe = await deployer.deploy(GnosisSafe);
+    const owners = [johnAddress];
```

Comment:
Should we still call them `john`?

Reply:
Oh, I'm not replying same name for all tests, should I? I'm trying to keep the wallet's names most clear as possible for each file/test suite.
Will be better if I called it as `signer` ?

Reply:
What do you mean by "replying same name"?

Reply:
Keeping the wallet's name as clear as possible for each file / test suite sounds good.

Sure, `signer` sounds like a good name

Reply:
Using the same name to accounts among test suites.

Reply:
Done.

Reply:
Ah ok - yeah I think the accounts can be used for different roles between different files / test suites

Reply:
Resolving this